tests/unit/amo/components/TestHeroRecommendation.js

```diff
@@ -91,6 +91,32 @@ describe(__filename, () => {
         root.instance().makeCallToActionURL(),
       );
     });
+
+    it('configures an external link to open in a new tab', () => {
+      const _isInternalURL = sinon.stub().returns(false);
+      const shelfData = createShelfData({
+        external: fakePrimaryHeroShelfExternal,
```

Comment:
Since this relies on a stub, I suggest asserting that `fakePrimaryHeroShelfExternal` gets passed as a parameter to `_isInternalURL()`.

Reply:
This is a good idea. `_isInternalURL()` is just passed `this.makeCallToActionURL()` in the code, but I gather you don't want me to just assert that it's called with that, but rather with whatever the result of `this.makeCallToActionURL()` would be, in this case. I've done that using a sinon matcher which I think gives us what we want, but if you could take another look at it that would be great.